### ISSUE TITLE
Congenital amputation 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10960]
**Preferred term label:**

**Synonyms**
intra-uterine amputation

**Definition (free text, please give PubMed ID)**
PMID: 31424867. There is no known clear etiology of ABS. However, it is thought that rupture of amnion early in pregnancy results in the development of multiple loose strands (amniotic bands) that adhere to or entangle the fetus. Constriction of otherwise normally developing organs results in constriction rings, some severe cases leading to vascular disruption and could potentially result in **amputation of the involved anatomic structure**. 
(...) Constriction rings and **limb or digital amputation** are the most common findings, present in at least 80 percent of cases. The amniotic bands may be present, confined to the skin or soft tissue, and extend deep into the tissue. If constriction results in **amputation in utero**, then the amputated part is usually resorbed and not visible following birth.

PMID: 31407870 The phenotypic presentation in the extremities has considerable variation which can range from simple constriction rings to more complex scenarios. This includes circumferential constriction rings, which can lead to distal ischemia resulting in significant edema and/or limb and digital amputations.

PMID: 25531236 Amniotic band syndrome (ABS) encompasses a wide spectrum of disease of uncertain etiology. The presentation of the disease is dependent on the anatomic location of the bands, where bands affecting the extremities can lead to amputation and pseudosyndactyly, while broad bands can result in fatal conditions such as the limb-body wall complex (LBWC)

**Parent term (use hpo.jax.org/app)**
Abnormal fetal skeletal morphology HP:0025662

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 217100 CONSTRICTING BANDS, CONGENITAL
ORPHA:295000 Amniotic band syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001389
- label: Intrauterine amputation
- parents: [Abnormal fetal morphology (HP:0034058)](https://hpo.jax.org/app/browse/term/HP:0034058)
- pmid: PMID:40818398|PMID:25531236|PMID:31407870
- def: A developmental anomaly characterized by partial or complete absence of a limb or digit at birth, resulting from interruption of normal limb development in utero.
- comment: The two main mechanisms involve amniotic band disruption (constriction of a developing limb or digit by fibrous amniotic bands) and vascular disruption (e.g., thrombosis, hemorrhage, or ischemia).
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10960
- synonyms: Congenital amputation(NONE)|Intra-uterine amputation(NONE)
